### ISSUE TITLE
Godot element focus ordering not visible to VoiceOver screenreader

### ISSUE THREAD

**Issue by PeterDrake** [godotengine/godot#120247]
### Tested versions

Reproducible in 4.6.3.stable

### System information

Godot v4.6.3.stable - macOS Sequoia (15.7.0) - Multi-window, 1 monitor - OpenGL 3 (Compatibility) - Apple M1 - Apple M1 (8 threads) - 16.00 GiB memory

### Issue description

The next focus element in a Control node can be set in the Inspector. This affects tab and arrow keys, but the "next element" key in macOS VoiceOver screenreader does not recognize it.

### Steps to reproduce

Using the minimal reproduction project below:

1. Make and run a Mac build.
2. Notice that, from button 1, either right arrow, down arrow, or tab goes right to button 3.
3. Navigate back to button 1 and start VoiceOver. Wait for it to recognize and read button 1.
4. Press ctrl-option-space. Instead of button 3, it goes to button 2.

### Minimal reproduction project (MRP)

This tiny project demonstrates the problem on commit aec4f5:

https://github.com/PeterDrake/screenreader

**Comment by bruvzg** [comment]
Not sure if there's any way to specify element order using AccessKit, there were properties for this, but were never functional and were removed (https://github.com/AccessKit/accesskit/pull/288).

**Comment by PeterDrake** [comment]
Certainly a screenreader accessing a web page allows the user to navigate between groups and then descend into elements within them. Is there no way to get even that level of control? I'm new to screenreaders, but I find that surprising.

**Comment by PeterDrake** [comment]
Try commit f5e074. It has two groups of buttons. Initially, the left group has the focus, which VoiceOver recognizes. When I hit the VO key for "next element", I would expect it to move on to the sibling in the node tree (the right group). Instead, it moves to the first button _within_ the right group.

Is there any way for Godot to provide the "structure" (even if not the ordering) of elements for a screenreader?

**Comment by bruvzg** [comment]
Structure is always the same as node tree, but containers are ignored by default. To make container work as a group, enable `Accessibilty Region` property.

**Comment by PeterDrake** [comment]
I don't see `Accessibility Region` under `Accessibility` and code auto-complete doesn't recognize it. How do I set it?

**Comment by PeterDrake** [comment]
Ah, it looks like this is a beta feature that isn't in the stable version (4.6) yet.

**Comment by PeterDrake** [comment]
Downloaded 4.7_rc3 and it works!
